src/main/java/org/cactoos/collection/CollectionEnvelope.java

```diff
@@ -23,6 +23,7 @@
  */
 package org.cactoos.collection;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
@ilyakharlamov I didn't like this kind of dependency. It's a third party library into cactoos and should be removed.